src/amp.cc

```diff
@@ -0,0 +1,53 @@
+#include "../include/amp/amp.hh"
+#include <cmath>
+
+Audio Amplifier::amplify(Audio &sample, double amount) {
```

Comment:
Since this has no state, you should make it static

Reply:
This will also allow you to test it without instantiating an object

Reply:
I believe so, with modern C++ you only have to tell it in the header